build/release-notes.md

```diff
@@ -8,6 +8,127 @@ toc_max: 2
+  it can be shown in provenance for later debugging. Previously this only
+  happened when building from a Git URL directly. To opt-out of this behavior
+  you can set `BUILDX_GIT_INFO=0`. Optionally you can also automatically define
+  labels with VCS info by setting `BUILDX_GIT_LABELS=1`. {% include github_issue.md repo="docker/buildx" number="1462" %}
+  {% include github_issue.md repo="docker/buildx" number="1297" %} {% include github_issue.md repo="docker/buildx" number="1341" %}
+  {% include github_issue.md repo="docker/buildx" number="1468" %} {% include github_issue.md repo="docker/buildx" number="1477" %}
+* Named contexts with `--build-context` now support `oci-layout://` protocol
+  for initializing the context with a value of a local OCI layout directory.
+  E.g. `--build-context stagename=oci-layout://path/to/dir`. This feature
+  requires BuildKit v0.11.0+ and Dockerfile 1.5.0+. {% include github_issue.md repo="docker/buildx" number="1456" %}
+* Bake now supports [resource interpolation](bake/configuring-build.md#resource-interpolation)
+  where you can reuse the values from other target definitions. {% include github_issue.md repo="docker/buildx" number="1434" %}
+* Buildx will now automatically forward `SOURCE_DATE_EPOCH` build-arg if one is
+  defined in your environment. This feature is meant to be used with updated
+  reproducible builds support in BuildKit v0.11.0+. {% include github_issue.md repo="docker/buildx" number="1482" %}
+* Buildx will now remember the last activity for a builder for better
+  organization of builder instances. {% include github_issue.md repo="docker/buildx" number="1439" %}
+* Bake now supports [null values](bake/file-definition.md#null-values) for
+  build arguments and labels to use the defaults set in the Dockerfile {% include github_issue.md repo="docker/buildx" number="1449" %}
+* [`imagetools inspect` command](../engine/reference/commandline/buildx_imagetools_inspect.md)
+  now supports showing SBOM and Provenance data {% include github_issue.md repo="docker/buildx" number="1444" %}
+  {% include github_issue.md repo="docker/buildx" number="1498" %}
+* Increase performance of `ls` command and inspect flows {% include github_issue.md repo="docker/buildx" number="1430" %}
+  {% include github_issue.md repo="docker/buildx" number="1454" %} {% include github_issue.md repo="docker/buildx" number="1455" %} 
+  {% include github_issue.md repo="docker/buildx" number="1345" %}
+* Adding extra hosts with [Docker driver](drivers/docker.md) now supports
+  Docker specific "host-gateway" special value {% include github_issue.md repo="docker/buildx" number="1446" %}
+* [OCI exporter](exporters/oci-docker.md) now supports `tar=false` option for
+  exporting OCI format directly in a directory {% include github_issue.md repo="docker/buildx" number="1420" %}
+
+### Updates
+
+* Updated the Compose Specification to 1.6.0 {% include github_issue.md repo="docker/buildx" number="1387" %}
```

Comment:
How about upgrades instead of updates? Did this for Engine v23

```suggestion
### Upgrades

* Upgrade the Compose Specification to 1.6.0 {% include github_issue.md repo="docker/buildx" number="1387" %}
```

Reply:
Not against it I was just following the current layout as we are using `### Updates` since https://github.com/docker/docs/pull/16208